internal/protoschema/jsonschema/jsonschema.go

```diff
@@ -86,6 +86,13 @@ func (p *jsonSchemaGenerator) getID(desc protoreflect.Descriptor) string {
 	return string(desc.FullName()) + ".schema.json"
 }
 
+func (p *jsonSchemaGenerator) getRef(fdesc protoreflect.FieldDescriptor) string {
+	if fdesc.Parent() == fdesc.Message() {
+		return "#"
+	}
```

Comment:
Note that this only supports recursion of depth 1. Is there more planned here?

Reply:
If it's more than one, it used a normal file reference

Reply:
Yeah, mainly wanted to confirm that's still the case. If/when we implement single-file support, we'll want to come back here.